omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/EncounterTypeController1_8Test.java

```diff
@@ -131,6 +135,25 @@ public void shouldRetireAEncounterType() throws Exception {
 		assertEquals(reason, service.getEncounterTypeByUuid(getUuid()).getRetireReason());
 	}
 	
+	@Test
+	public void shouldUnRetireAnEncounterType() throws Exception {
+		EncounterType encounterType = service.getEncounterTypeByUuid(getUuid());
+		encounterType.setRetired(true);
+		encounterType.setRetiredBy(Context.getAuthenticatedUser());
```

Comment:
I guess you do not need to set retiredBy, retireReason, and dateRetired.

Reply:
Yes, thank you for pointing out this @dkayiwa , these methods are deprecated apart from `setRetireReason()`.